src_c/mask.c

```diff
@@ -1360,8 +1370,10 @@ mask_get_bounding_rects(PyObject *self, PyObject *args)
     }
 
     ret = PyList_New(0);
-    if (!ret)
+    if (!ret) {
+        free(regions);
         return NULL;
+    }
```

Comment:
I noticed another issue below this, with PyList_Append below not being checked for failure. https://docs.python.org/3.7/c-api/list.html#c.PyList_Append

With pgRect_New4 also.

Reply:
I made an issue for the PyList_Append problems here: https://github.com/pygame/pygame/issues/1332